Sources/Kevin/Constants/Constants.swift

```diff
@@ -10,5 +10,5 @@
 import Foundation
 
 struct Constants {
-    static let apiVersion = "2.4.1"
+    static let apiVersion = "2.5.0"
```

Comment:
Just a question: Maybe we should rename this constant? I was confused for a second as I thought it is related to API version SDK is using.

Reply:
Yeah, `skdVersion` would make much more sense